platform/security/rest/security-rest-cxfwrapper/src/main/java/org/codice/ddf/cxf/paos/PaosInInterceptor.java

```diff
@@ -286,7 +288,7 @@ private String createToken(String authorization) throws IOException {
 
     HttpResponseWrapper getHttpResponse(String responseConsumerURL, String soapResponse)
             throws IOException {
-        HttpTransport httpTransport = new ApacheHttpTransport();
+        HttpTransport httpTransport = new NetHttpTransport();
```

Comment:
Should we add a comment about why we need this particular implementation for this to work?